code/modules/organs/limbs.dm

```diff
@@ -927,6 +927,9 @@ This function completely restores a damaged organ to perfect condition.
 			SPAN_HIGHDANGER("<b>Your [display_name] goes flying off!</b>"),
 			SPAN_WARNING("You hear a terrible sound of ripping tendons and flesh!"), 3)
 
+			if(body_part != BODY_FLAG_HEAD)
+				owner.emote("scream")
```

Comment:
this'll need an invoke_async, ask one of the devs for advice on discord to implement this if you get stuck, but try doing it yourself first